rpc/filters/api.go

```diff
@@ -67,6 +67,7 @@ func (api *PublicFilterAPI) isEth() bool {
 func (api *PublicFilterAPI) timeoutLoop() {
 	// TODO ek – infinite loop; add shutdown/cleanup logic
 	ticker := time.NewTicker(5 * time.Minute)
+	defer ticker.Stop()
```

Comment:
Do you think this can be the potential memory leak reason?

Reply:
I just corret them all, Some of this code will not have memory leaks because it is an infinite loop and does not exit.